e2e/playground.py

```diff
@@ -176,24 +154,40 @@ def expect_class_value(
     timeout: Timeout = None,
 ) -> None:
     """Expect a locator to have (or not to have) a class value"""
-    cls_regex = re.compile(rf"\b{cls}\b")
+    cls_regex = re.compile(rf"\b{re.escape(cls)}\b")
```

Comment:
If `cls` is `"foo"`, then this regex will match `"foo-bar"`, which is not what you want.